SickDesignSystem/synergy
# feat: ✨ React component support

# Pull Request

## 📖 Description

This PR provides a new package named `@synergy-design-system/react` that makes it possible to use the web components from `@synergy-design-system/components` in React.js based applications in an easy way.

### 🎫 Issues

Closes #43
Closes #62

## 👩‍💻 Reviewer Notes

- There is currently no semantic-release support. This has to be done in another ticket.
- Please also read the provided `README.md` to make sure I have not forgotten anything :)
- I refactored the original build system to a more streamlined and modular system. I think this better fits our needs than the original system from components. Looking forward for your opinions :)

## 📑 Test Plan

- Check if all scripts work as intended.
- Check if the provided demo works.
- Play around with the demo
